go-controller/pkg/ovn/ovn.go

```diff
@@ -271,6 +287,16 @@ func (oc *DefaultNetworkController) removeRemoteZonePod(pod *kapi.Pod) error {
 				getPodNamespacedName(pod), err)
 		}
 	}
+	if kubevirt.IsPodLiveMigratable(pod) {
+		// After live migration to a different zone ip should be deallocated
+		// from remote zone if VM is gone, not that using deleteLogicalPort to
+		// de-allocate the IPs should not fail even if the port do not exist
+		// since it's idempotent.
+		if err := oc.deleteLogicalPort(pod, portInfo); err != nil {
```

Comment:
Perhaps we could check with the suggested `zoneContainsPodSubnet` first.

It's idempotent but it still cost cycles both in ovn-k and nbdb.

Reply:
I am going for the gold and calling just the part that matters here.